feedback/invalid_token.go

```diff
@@ -213,16 +221,38 @@ func (i *InvalidTokenHandler) deleteTokensFromGame(tokens []string, game, platfo
 	query := queryBuild.String()
 
 	l.Debug("deleting tokens")
-	_, err := i.Client.DB.Exec(query, params...)
+	res, err := i.Client.DB.Exec(query, params...)
 	if err != nil && err.Error() != "pg: no rows in result set" {
 		raven.CaptureError(err, map[string]string{
 			"version": util.Version,
 			"handler": "invalidToken",
 		})
 
 		l.WithError(err).Error("error deleting tokens")
+		statsReporterReportMetricCount(i.StatsReporter,
+			MetricsTokensDeleteError, int64(len(tokens)), game, platform)
+
 		return err
 	}
 
+	if err != nil && err.Error() == "pg: no rows in result set" {
+		statsReporterReportMetricCount(i.StatsReporter,
+			MetricsTokensDeleteNonexistent, int64(len(tokens)),
+			game, platform)
+
+		return nil
+	}
+
+	if res.RowsAffected() != len(tokens) {
+		statsReporterReportMetricCount(i.StatsReporter,
+			MetricsTokensDeleteNonexistent, int64(len(tokens)-res.RowsAffected()),
+			game, platform)
+
+		return nil
```

Comment:
Here we should also report MetricsTokensDeleteSuccess with count `res.RowsAffected()` before returning, right?

Reply:
Yep, you're right! I'll change that.